### ISSUE TITLE
mmenu dropdown bootstrap 4 wrapper issue

### ISSUE THREAD

**Issue by reiuz** [FrDH/jQuery.mmenu#924]
Hi guys
We have a problem with the bootstrap 4 wrapper:
we need to have a single link button for a second level sub-menu; even trying use the span tag as suggested in your documentation what we obtain is a unclickable label and only the arrow link is opening the sub-menu. 

Here's the code:

```
<ul class="navbar-nav mr-auto">
. . .
   <li class="nav-item dropdown">
              <span class="nav-link">Label1</span>
              <ul class="dropdown-menu" id="dropdown-world" aria-labelledby="how">
                      <li class="dropdown-item active"><a class="nav-link">test</a></li>
                       <li class="dropdown-item active"><a class="nav-link">test2</a></li>
              </ul>
   </li>
. . .
</ul>
```

Thank you guys

**Comment by reiuz** [comment]
We are not using the a tag. 
What we are trying to do is to use the span tag as dropdown as specified in the page "Tips and tricks", section "Open submenus" in your website.
The effect we have following your example is that the label part of the menu is unclickable and only the arrow part is opening the sub-menu.

**Comment by FrDH** [comment]
Bootstrap uses the classnames `dropdown-toggle`, `dropdown-menu` and `dropdown-item`, the mmenu wrapper for Bootstrap searches for these classnames.
In your case I think your `<span>` is missing the `dropdown-toggle` classname.

**Comment by AHQY** [comment]
I had the same problem. I don't see any reference to "dropdown-toggle" in the bootstrap 4 wrapper. A quick fix is to update jquery.mmenu.bootstrap4.js:

`var a=n("<a />"),`

to:

`var a=e.hasClass("dropdown-toggle")?n("<span />"):n("<a />"),`

This looks for the "dropdown-toggle" class and uses a `<span>` rather than `<a>` element to wrap the list item. You can continue to use `<a>` for the Bootstrap "nav-link" element rather than having to use a `<span>` which is not typically used as a "nav-link" element wrapper in the Bootstrap docs. You will need to add the "dropdown-toggle" class to the "nav-link" element as FrDH suggested, for example:

`<a href="#" class="nav-link dropdown-toggle">Label1</a>`

If you are using the source script rather than the minified distribution script you can change:

`var $i = $('<a />');`

to:

`var $i = $a.hasClass('dropdown-toggle') ? $('<span />') : $('<a />');`

Which can be found in cloneLink().

**Comment by FrDH** [comment]
> If you are using the source script rather than the minified distribution script
The source is in Typescript, so that won't work :)

I'll implement this change in a future release.

**Comment by FrDH** [comment]
should be fixed in version 7.3
